File: src/main/java/org/example/bankramenserver/global/util/MoneyFormatter.java
package org.example.bankramenserver.global.util;

import java.text.NumberFormat;
import java.util.Locale;

public class MoneyFormatter {

    private static final NumberFormat FORMAT =
            NumberFormat.getNumberInstance(Locale.KOREA);

    public static String format(Long amount) {
        if (amount == null) {
            return "0";
        }
        return FORMAT.format(amount);
    }
}

File: src/main/kotlin/org/example/balogserver/global/util/MoneyFormatter.kt
package org.example.balogserver.global.util

import java.text.NumberFormat
import java.util.Locale

object MoneyFormatter { private val format = NumberFormat.getNumberInstance(Locale.KOREA); @JvmStatic fun format(amount: Long?) = amount?.let(format::format) ?: "0" }
